### DISCUSSION TITLE
Add Image for SEO

### DISCUSSION THREAD

**Discussion by AidaIssayeva** [razonyang/hugo-theme-bootstrap#167] [Ideas]
Hi @razonyang! 
I noticed when sharing a post, the image associated with the post doesn't get added in SEO. 

This is the example of the blog post with the blue image: 
<img width="500" alt="Screen Shot 2021-08-21 at 4 28 12 PM" src="https://user-images.githubusercontent.com/15741631/130334130-1148a692-2888-4287-a9c0-0dcc051231c4.png">
 
and this is on Twitter without any image: 
<img width="500" alt="Screen Shot 2021-08-21 at 4 27 59 PM" src="https://user-images.githubusercontent.com/15741631/130334147-3bb74ac4-52bb-4480-9c71-f7a55571d277.png">

It would be great to have the image of the post being added to SEO. 

Thank you!

**Comment by razonyang** [comment]
Thanks for pointing this out. That is because the example site does not specify the `baseURL`. It has been fixed by 2746d53.
You will be need to replace the `baseURL` with your site URL in `/config/default/production/config.toml`:

```
baseURL = "https://mysite.tld"
```

> It's no need to upgrade the theme.

**Reply by AidaIssayeva** [reply]
thank you for the prompt response!
I've updated the `baseURL` (`baseURL = "https://cupsofcode.com/"`) in `config/_default/config.toml` and `config/production/config.toml`, however, I still see the issue 💣  : 
<img width="500" alt="Screen Shot 2021-08-22 at 6 40 37 PM" src="https://user-images.githubusercontent.com/15741631/130372288-53ac56eb-2b0a-495b-802b-86b097d60496.png">

**Reply by razonyang** [reply]
@AidaIssayeva It works, I think this may be due to the cache problem of twitter card.
![image](https://user-images.githubusercontent.com/17720932/130377414-f90d73f4-72fb-42f5-8eb8-8ae821ffe3f0.png)
![image](https://user-images.githubusercontent.com/17720932/130377712-ba1e04b6-aaf2-4997-85a9-fe71705ae630.png)

**Comment by razonyang** [comment]
@AidaIssayeva FYI, I noticed that the cover image takes up a lot of space if the summary of the post is too long.
![image](https://user-images.githubusercontent.com/17720932/130422490-8c73c2a9-3030-4bc0-8b9e-7f2e5e8ae48d.png)

So I make some minor changes(the latest commit).

**Desktop**:

![image](https://user-images.githubusercontent.com/17720932/130420557-b5ce483c-98ce-4e21-982c-25ec40f2d4ef.png)

**Mobile**:

![image](https://user-images.githubusercontent.com/17720932/130420694-ff18af2d-ad20-4dd1-b915-a19f3b26cbdc.png)

**Reply by AidaIssayeva** [reply]
@razonyang I love it 😍 ! Looks much better! Thank you so much for this addition 🙏 !

**Reply by razonyang** [reply]
It's my pleasure.
